packages/core/src/modules/routing/services/MediationRecipientService.ts

```diff
@@ -69,6 +69,31 @@ export class MediationRecipientService {
     this.messageSender = messageSender
   }
 
+  public async requestStatus(
+    config: {
```

Comment:
`config` can be `config?` to avoid assigning a default empty object to it.

Reply:
This code was deprecated, my branch being based on an old branch brought it back in, I have removed this section.